src/parser/shared/modules/racials/bloodelf/ArcaneTorrent.js

```diff
@@ -14,17 +15,22 @@ class ArcaneTorrent extends Analyzer {
   constructor(options) {
     super(options);
     this.active = this.selectedCombatant.race && this.selectedCombatant.race === RACES.BloodElf;
+    
     if (!this.active) {
       return;
     }
+
+    const specID = this.selectedCombatant._combatantInfo.specID;
+    const isRogue = (specID === SPECS.ASSASSINATION_ROGUE.id || specID === SPECS.OUTLAW_ROGUE.id || specID === SPECS.SUBTLETY_ROGUE.id);
```

Comment:
If anyone knows a better way to check the class of the selected combatant please let me know.